tests/common/TestProjects/SystemMemoryReference/Program.cs

```diff
@@ -1,13 +1,13 @@
-﻿using System;
-
-namespace app
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine (typeof(ReadOnlySpan<char>).Assembly.FullName);
-            System.Console.WriteLine (typeof (UIKit.UIWindow));
-        }
-    }
-}
+using System;
+
+namespace app
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine (typeof(ReadOnlySpan<char>).Assembly.FullName);
+            System.Console.WriteLine (typeof (UIKit.UIWindow));
+        }
+    }
+}
```

Comment:
not clear why this file is changing for xcode12.5

Reply:
Yes looks like git got 🍌s here since there is really no actual change, same for SystemMemoryReference.csproj :/